gradle-sls-packaging/src/test/groovy/com/palantir/gradle/dist/service/ServiceDistributionPluginTests.groovy

```diff
@@ -104,12 +104,12 @@ class ServiceDistributionPluginTests extends GradleTestSpec {
 
         then:
         def version02BuildOutput = runSuccessfully(':build', ':distTar', ':untar02').output
-        version02BuildOutput ==~ /(?m)(?s).*^:createCheckScript UP-TO-DATE$.*/
-        version02BuildOutput ==~ /(?m)(?s).*^:createInitScript UP-TO-DATE$.*/
-        version02BuildOutput ==~ /(?m)(?s).*^:createLaunchConfig$.*/
-        version02BuildOutput ==~ /(?m)(?s).*^:createManifest$.*/
-        version02BuildOutput ==~ /(?m)(?s).*^:manifestClasspathJar$.*/
-        version02BuildOutput ==~ /(?m)(?s).*^:distTar$.*/
+        version02BuildOutput ==~ /(?m)(?s).*:createCheckScript UP-TO-DATE.*/
```

Comment:
the output in gradle 4.7 looks like:
```
> Task :createCheckScript UP-TO-DATE
```

honestly, we should be using nebula test (or similar) which lets you write stuff like:
```
result.wasUpToDate('createCheckScript')
```

but that's a change for another day..